src/pages/guides/tutorials/grids-addon.md

```diff
@@ -0,0 +1,1321 @@
+
+```js
+// ...
+runtime.exposeApi({
+	addGrid({ columns, rows, gutter, columnColor, rowColor }) {
+		const doc = editor.documentRoot;
+		const page = doc.pages.first;
+		const rowWidth = page.width;
+	  const rowHeight = (page.height - (rowsNumber + 1) * gutter) / rowsNumber;
+	},
+});
+```
+
+To draw all four of them (or any number coming from the UI) at once, a loop is in order.
+
+```js
+  // ...
+  var rowsRect = [];
+  for (let i = 0; i < rows; i++) {
+	  let r = editor.createRectangle();
+	  r.width = page.width;
+	  r.height = rowHeight;
+		// moving the row in place
+    r.translateY = gutter + (gutter + rowHeight) * i;
+    rowsRect.push(r);
+  }
+  // adding the rows to the page
+  rowsRect.forEach((rect) => page.artboards.first.children.append(rect));
+```
+
+We've created all the needed rectangles, shifting them on the Y-axis according to their number and gutter, collecting them in a `rowsRect` array; which, in turn, we've looped through to append them all to the first `artboard` in the page.
+
+<!-- code below -->
+<InlineAlert variant="info" slots="text1" />
+
+Please remember that you must always `append()` elements to their container's `children` property.
+
+Using the same principles, we can create columns: rectangles as tall as the page and whose width we can compute.
+
+```js
+const doc = editor.documentRoot;
+const page = doc.pages.first;
+var colsRect = [];
+const colWidth = (page.width - (cols + 1) * gutter) / cols;
+for (let i = 0; i < cols; i++) {
+	let r = editor.createRectangle();
+	  r.width = colWidth;
+	  r.height = page.height;
+		r.translateX = gutter + (gutter + colWidth) * i;
+	cols.push(r);
+}
+cols.forEach((rect) => page.artboards.first.children.append(rect));
+```
+
+We now have most of what is needed to complete the Grids add-on; we're in dire need of a better structure, though.
+
+### Organizing the code
+
+The Grid creation process can be split into **smaller, separate steps**—we can take this chance to think about how to structure the entire project.
+
+- Creating a rectangle is obviously best dealt with with a dedicated `createRect()` function.
+- Rows and Columns can be separate processes, too.
+- `code.js` doesn't need to expose anything else but the `addGrid()` and  `deleteGrid()` methods.
+- `addRows()` and `addColumns()` can belong to the `shapeUtils.js` module and imported in `script/code.js`, while `createRect()` will be kept as private.
+- The Rows and Columns colors come from the UI as hex strings, like `"#ffcccc"`; we must write a function that extracts R, G, and B values and maps them to the (0..1) range the Document API deals with.
```

Comment:
This should get easier when we expose some more methods re: color in the API soon.